core/shared/src/main/scala/caseapp/core/util/NameOps.scala

```diff
@@ -4,36 +4,38 @@ import scala.language.implicitConversions
 
 import caseapp.Name
 
-class NameOps(val name: Name) extends AnyVal {
+class NameOps(val name: Name) {
```

Comment:
Why removing the `AnyVal`?

Reply:
😓 yeah this was part of an experimental change...